modules/kubernetes-addons/kyverno/kyverno-ui-values.yaml

```diff
@@ -0,0 +1,11 @@
+ui:
+  enabled: true
+
+kyvernoPlugin:
```

Comment:
remove all of this and the values file in general - users can decide to use the kyverno addon or the plugin